Gem/Code/Source/DecalExampleComponent.cpp

```diff
@@ -158,4 +182,14 @@ namespace AtomSampleViewer
         m_decalContainerClone = AZStd::make_unique<DecalContainer>(decalFeatureProcessor, AZ::Vector3(-1,0,0));
     }
 
+    void DecalExampleComponent::UpdateDirectionalLight()
+    {
+        using namespace AZ;
+
+        constexpr float directionalLightDist = 10.0f;
```

Comment:
nit: use this instead of the 10.0?

Reply:
@Adi-Amazon Hi Adi. I missed your comment until just now. Could you clarify what you meant?